src/world/physicsReferences.lua

```diff
@@ -76,6 +76,16 @@ function PhysicsReferences.getCategory(type)
 end
 
 function PhysicsReferences.setFixtureType(fixture, type)
+	if categories[type] == nil then
```

Comment:
This can be one if statement. nil should be NAN. I forget the proper test for that it would be something like:
type(*table*[type]) ~= "number"

Reply:
I'm not sure. If the type is not present (that is, it's nil) then there's a bug in the user's code. If the type is present but not a number, then there's a bug in PhysicsReferences. That's something that we should catch in our tests, but not necessarily cause an error at runtime.